docs/ops/image/Interpolate_4.md

```diff
@@ -0,0 +1,454 @@
+## Interpolate <a name="Interpolate"></a>
+
+**Versioned name**: *Interpolate-4*
+
+**Category**: Image processing
+
+**Short description**: *Interpolate* layer performs interpolation of independent slices in input tensor by specified dimensions and attributes.
+
+**Attributes**
+
+* *axes*
+
+  * **Description**: `axes` specify dimension indices where interpolation is applied, and `axes` is any unordered list of indices of different dimensions of input tensor, e.g. `[0, 4]`, `[4, 0]`, `[4, 2, 1]`, `[1, 2, 3]`. These indices should be non-negative integers from `0` to `rank(data) - 1` inclusively.  Other dimensions do not change. The order of elements in `axes` attribute matters, and mapped directly to elements in the 2nd input `target_spatial_shape`. Namely, `output_shape[axes[i]] = target_spatial_shape[i]` for all `i in range(0, len(axes))` and `output_shape[j] = input_shape[j] + pads_begin[j] + pads_end[j]` for `j not in axes`, `j in range(0, rank(data))`.
```

Comment:
Make it an input.

Reply:
Done.

Reply:
ONNX [Resize ](https://github.com/onnx/onnx/blob/master/docs/Operators.md#resize) requires to specify scale or size for each of the data input dimensions, so interpolate `axes` is always like `0...(n-1)` where `n` is a number of elements in `target_spatial_shape` and equal to data rank. 
If `axes` is required input, we will have to add in onnx importer some operations like `shape_of` and `range` to provide the `axes`.
 
What do you think about making interpolate `axes` optional with `0...(n-1)` indices as default?